### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by AvnerMaster**
Added:
Readme.md
Setup.md
.env example

### CODE REVIEW

File: quickstart-node/README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # Monday-Code NodeJS Quickstart
 
-This Node - express with typescript application is designed to showcase the practical use of the monday code platform by integrating various monday app functionalities.
-It's an essential resource for developers looking to leverage Monday Code features through NodeJS environment.
+This Node - express with typescript application is designed to showcase the practical use of monday code and monday api
+sdk by
```
Comment (oricho123) [inline quickstart-node/README.md:4]: Weird newline

File: quickstart-node/README.md [hunk 2]
```diff
@@ -26,6 +30,7 @@ npm i
 ```
 
 ## Compiling Files
```
Comment (oricho123) [inline quickstart-node/README.md:32]: Is the compiling and running sections true? is this how you've done it?

**Review by oricho123 — APPROVED** [review]
Amazing work man! @yuhgto   FYI

File: quickstart-node/README.md (hunk 2 above)
Comment (AvnerMaster) [reply quickstart-node/README.md:32]: Yes, this is the script property in the package.json:
`"scripts": {
    "build": "tsc",
    "start": "node -r ts-node/register/transpile-only -r tsconfig-paths/register dist/index.js",
    "dev": "nodemon",
    "fetch:schema": "bash fetch-schema.sh",
    "codegen": "graphql-codegen",
    "fetch:generate": "npm run fetch:schema && npm run codegen"
  },`

File: quickstart-node/README.md [hunk 3]
```diff
@@ -50,4 +53,121 @@ If you did not set a port environment variable, the application will start runni
...
+* Using Monday apps Queue
+* Using Monday code Logger
+
+**NOTE**: Storage feature does not work locally, you can only check it after you push your code to Monday Code
+
+### Monday API SDK Usage
+
+* Example of updating column value in an item
+* Example of a custom query with types generate
+* Example of monday api error handling
```
Comment (RomKadria) [inline quickstart-node/README.md:71]: grammer please :) 
An example of updating a column value in an item.
An example of a custom query with generated types.
An example of error handling with the Monday API.

File: quickstart-node/README.md [hunk 4]
```diff
@@ -50,4 +53,121 @@ If you did not set a port environment variable, the application will start runni
...
+### Custom Actions and Triggers in Monday
+
+* Creating custom actions and triggers
+* Implementing an integration recipe
+* Deploying the app to Monday code
+
+### Error Handling
+
+* Example of global error handler middleware
+* Example of Monday API error handler middleware
```
Comment (RomKadria) [inline quickstart-node/README.md:87]:
```suggestion
* Example of Monday API error handling middleware
```

File: quickstart-node/README.md [hunk 5]
```diff
@@ -50,4 +53,121 @@ If you did not set a port environment variable, the application will start runni
...
+    participant Monday SDK API
+
+    User->>Monday: Adds recipe from automation center
+    Monday->>App: Sends request to app's authorization URL
+    App->>User: Redirects user to Monday's OAuth authorization page (if not yet authorized)
+    User->>Monday: Logs in and authorizes the app
+    Monday->>App: Sends user back to app's callback URL with an authorization code
+    App->>Monday: Requests access token using the authorization code
+    Monday->>App: Sends access token
+    App->>Monday SDK API: Saves the access token using StorageAPI
```
Comment (RomKadria) [inline quickstart-node/README.md:123]: should be used as monday api sdk (in all places) and not the sdk first =)

Comment (RomKadria) [reply quickstart-node/README.md:123]: also this is more of a client also. i wouldve called it just monday api

**Review by RomKadria — APPROVED** [review]

File: quickstart-node/README.md [hunk 6]
```diff
@@ -1,11 +1,13 @@
 # Monday-Code NodeJS Quickstart
 
-This Node - express with typescript application is designed to showcase the practical use of the monday code platform by integrating various monday app functionalities.
-It's an essential resource for developers looking to leverage Monday Code features through NodeJS environment.
+This Node - express with typescript application is designed to showcase the practical use of monday code and monday api
+sdk by integrating various monday app functionalities. It's an essential resource for developers looking to leverage
+Monday Code and Monday API features through NodeJS environment.
 
 ## Getting Started
```
Comment (yuhgto) [inline quickstart-node/README.md:7]: Sorry one more thing – IMO you should use the monday apps tunnel in your `npm run dev` script... 

Check our other quickstarts for an example - https://github.com/mondaycom/welcome-apps/blob/2015c76296c6ecc36b7dbaa6559adc4758342757/apps/quickstart-integrations/package.json#L7

File: quickstart-node/README.md [hunk 7]
```diff
@@ -50,4 +53,49 @@ If you did not set a port environment variable, the application will start runni
...
+* Implementing OAuth process
+* Authenticating with JWT (JSON Web Tokens)
+
+### Custom Actions and Triggers in Monday
+
+* Creating custom actions and triggers
+* Implementing an integration recipe
+* Deploying the app to Monday code
+
+### Error Handling
```
[the remark this answers is not in the document]
Comment (AvnerMaster) [reply quickstart-node/README.md:84]: I think it is necessary when you create a module, but here we give an example application so I don't think it is really needed especially when the code is typed. But I did added comments in the code in the places where it is actually needed. For example [here](https://github.com/mondaycom/monday-code-quickstarts/blob/feat/avnerha/readme/quickstart-node/src/middlewares/monday-api.middleware.ts#L22)

File: quickstart-node/README.md [hunk 8]
```diff
@@ -1,11 +1,13 @@
 # Monday-Code NodeJS Quickstart
 
-This Node - express with typescript application is designed to showcase the practical use of the monday code platform by integrating various monday app functionalities.
-It's an essential resource for developers looking to leverage Monday Code features through NodeJS environment.
+This Node - express with typescript application is designed to showcase the practical use of monday code and monday api
```
[the remark this answers is not in the document]
Comment (AvnerMaster) [reply quickstart-node/README.md:3]: The new app introduction is this now:

# Monday-Code NodeJS Quickstart

This Node - express with typescript application is designed to showcase the practical use of monday code and monday
platform API SDK. It also includes some routes examples that you can use as part of an automation.
You can use this app as an app feature, and edit it to your needs.

File: quickstart-node/SETUP.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
+## Setup Overview
```
[the remark this answers is not in the document]
Comment (AvnerMaster) [reply quickstart-node/SETUP.md:1]: The goal of this app is to show usage examples, not serve as a template. You can see it as like a welcome app. Its goal is to show developers how to use monday code, monday api and even create an automation with the routes examples in this app.

[the remark this answers is not in the document]
Comment (yuhgto) [reply quickstart-node/SETUP.md:1]: The quickstarts listed above are not really templates either, they are more like working examples. So I think it's a good fit for the template center.

File: quickstart-node/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (yuhgto) [reply quickstart-node/README.md:99]: works for me!

**Pull request merged**